src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.PackTool.targets

```diff
@@ -71,6 +74,15 @@ Copyright (c) .NET Foundation. All rights reserved.
   </Target>
 
   <Target Name="_PackToolValidation" Condition=" '$(PackAsTool)' == 'true' ">
+
+    <GetNuGetShortFolderName
```

Comment:
cc @zivkan Remember the extension point for pack that's supposed to help customers get the short name?
This might be it :)